src/ResourceManager/KeyVault/Commands.KeyVault/Commands/GetAzureKeyVaultKey.cs

```diff
@@ -58,6 +58,7 @@ public class GetAzureKeyVaultKey : KeyVaultCmdletBase
             HelpMessage = "Vault name. Cmdlet constructs the FQDN of a vault based on the name and currently selected environment.")]
 
         [ValidateNotNullOrEmpty]
+        [ValidatePattern(Constants.VaultNameRegExString)]
```

Comment:
Please be sure that this would not exclude any currently=-working values in scripts.